src/v/cloud_storage/cache_service.h

```diff
@@ -40,7 +40,6 @@ namespace cloud_storage {
 
 // These timeout/backoff settings are for S3 requests
 using namespace std::chrono_literals;
```

Comment:
>This pattern is not without precedent insomuch as many manifest downloads are subject to manifest_uplaod_timeout already.

I wonder if we've gotten feedback that this is confusing, and whether we should put in effort to make download-specific properties